modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -9,7 +9,7 @@ During installation, you can set the `networkConfig` configuration setting in th
 
 [IMPORTANT]
 ====
-Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes.
+Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes. *BZ#2048600* is a known issue where the bootstrap VM doesn't receive an IP address in the absence of a DHCP server. *BZ#2036677* is a known issue where the configured IP addresses get reset by the DHCP server, if present. Use the linked knowledge base articles in the release notes to avoid these issues.
```

Comment:
* doesn't -> does not
* I'd suggest linking to the bugzilla here - just listing it like this might not be clear to users what it is
* You can link the kbase here if you want (instead of sending them to the RNs for it); it's just xrefs that aren't allowed in modules.

Reply:
I see.